packages/@okta/vuepress-site/docs/guides/anything-as-a-source/index.md

```diff
@@ -0,0 +1,366 @@
+
+* using Okta Workflows
+
+   or
+
+* developing a custom client
+
+With either method, you need to first define your HR source in your Okta org. This is referred to as the Custom Identity Source integration. Okta provides you with a Custom Identity Source unique identifier that you can use in your Okta Workflow or custom client to identify the HR source. See Configuration of a Custom Identity Source in [Using anything as a source](https://okta.github.io/doc_reviews/en-us/Content/Topics/users-groups-profiles/usgp-anything-as-a-source.htm).
+
+This guide outlines the XaaS API flow so that you can develop your custom client for the XaaS integration. For XaaS integrations using Okta Workflows, see [Okta Workflows](https://help.okta.com/okta_help.htm?type=wf).
+
+## XaaS API concepts
+
+### Import Session
+
+The XaaS API synchronizing data flow uses an [Import Session](/docs/reference/xaas/#import-session) object to encapsulate the data upload and the data import processing tasks. You need to create an Import Session object each time you want to synchronize data from the HR source to Okta. The Import Session object uses the following `status` values to indicate each stage of the synchronization process flow.
+
+### Import Session status
+
+* **CREATED**: The Import Session object has been created for a specific Custom Identity Source integration and data can be loaded to the session at this stage. No data import processing has been invoked and the session can be cancelled at this stage.
+* **TRIGGERED**:  Okta is processing the loaded data in the Import Session. You can’t load new data to the Import Session object at this stage and the session can’t be cancelled. Import Sessions with this status can be viewed in the [Import Monitoring](https://help.okta.com/okta_help.htm?id=ext-view-import-monitoring-dashboard) page from the Admin Console.
+* **COMPLETED**:  The data in the Import Session object has been processed by Okta. You can’t upload new data to the Import Session object if it has the `COMPLETED` status. The synchronization data job is considered complete.
+* **CLOSED**: The session is cancelled and isn't available for further activity. Only Import Sessions with the `CREATED` status can be cancelled. You can't cancel a session that has been triggered or completed. Previously loaded user data is deleted from a cancelled Import Session.
+* **EXPIRED**: This status indicates that the Import Session has timed out during the data loading stage. An Import Session with the `CREATED` status expires from 24 hours of inactivity.
+
+### Import Session process
+
+You can only process one Import Session, for a specific Custom Identity Source integration, at a time to avoid conflicts. The following are additional Import Session behaviors:
```

Comment:
```suggestion
You can only process one Import Session at a time (for a specific Custom Identity Source integration) to avoid conflicts. The following are additional Import Session behaviors:
```